src/Components/Web.Extensions/src/ProtectedBrowserStorage/ProtectedBrowserStorage.cs

```diff
@@ -4,16 +4,19 @@
 using System;
 using System.Collections.Concurrent;
 using System.Runtime.InteropServices;
+using System.Runtime.Versioning;
 using System.Text.Json;
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.DataProtection;
 using Microsoft.JSInterop;
 
 namespace Microsoft.AspNetCore.Components.Web.Extensions
 {
+
     /// <summary>
     /// Provides mechanisms for storing and retrieving data in the browser storage.
     /// </summary>
+    [UnsupportedOSPlatform("browser")]
```

Comment:
@jeffhandley am I holding this right? Initializing this type throws a PlatformNotSupportedException in Browser:

https://github.com/dotnet/aspnetcore/blob/master/src/Components/Web.Extensions/src/ProtectedBrowserStorage/ProtectedBrowserStorage.cs#L34-L37

Reply:
Yep; looks good!